### DISCUSSION TITLE
Beta test: spaCy Curated Transformer pipelines

### DISCUSSION THREAD

**Discussion by danieldk** [explosion/spaCy#12848] [News & Announcements]
We have recently announced the first the [tech preview](https://github.com/explosion/curated-transformers/releases/tag/v0.9.0) of [Curated Transformers](https://github.com/explosion/curated-transformers). Curated Transformers is a transformer library for PyTorch that is developed by Explosion. It provides a curated set of state-of-the-art models that are composed of reusable building blocks.

Since Curated Transformers is a pure PyTorch library, we have also released a library with Thinc model wrappers and a spaCy pipeline component called `spacy-curated-transformers`. With this library, you can use Curated Transformers in your own pipelines. We have prepared an [example project](https://github.com/explosion/spacy-curated-transformers/tree/main/project) that shows how you can train a pipeline with Curated Transformers.

Today, we are releasing experimental spaCy transformer pipelines that use Curated Transformers. These pipelines are used when you download a transformer model using the spaCy 3.7.0 development release:

```bash
$ pip install spacy==3.7.0.dev
$ spacy download de_dep_news_trf
```

If you are using our transformer pipelines with spaCy 3.6 or earlier versions, it would be very helpful if you could test these new transformer pipelines and report significant regressions in speed or accuracy or for that matter, any other issues that you might encounter.

## Available Curated Transformer pipelines

| Model name       | Language  |
| ---------------- | --------- |
| ca_core_news_trf | Catalan   |
| da_core_news_trf | Danish    |
| de_dep_news_trf  | German    |
| en_core_web_trf  | English   |
| es_dep_news_trf  | Spanish   |
| fr_dep_news_trf  | French    |
| ja_core_news_trf | Japanese  |
| sl_core_news_trf | Slovenian |
| uk_core_news_trf | Ukrainian |
| zh_core_web_trf  | Chinese   |

**Comment by adrianeboyd** [comment]
The default pipelines with `spacy download` have just been updated to support `replace_listeners` (pipeline version with `spacy validate` should show `3.7.1`).
